.gitignore

```diff
@@ -1,4 +1,4 @@
 node_modules
 lib
-/index.js
-/stdlib.js
+/*.js
```

Comment:
I'd rather be explicit; what other files are you adding here that need to be gitignored?